full-node/sov-prover-storage-manager/src/dummy_storage.rs

```diff
@@ -0,0 +1,157 @@
+use std::marker::PhantomData;
+
+use byteorder::{BigEndian, ReadBytesExt};
+use sov_schema_db::schema::{KeyDecoder, KeyEncoder, Result as CodecResult, ValueCodec};
+use sov_schema_db::snapshot::{DbSnapshot, QueryManager};
+use sov_schema_db::{define_schema, CodecError};
+use sov_state::MerkleProofSpec;
+
+/// Oversimplified representation of [`sov_state::ProverStorage`]
```

Comment:
WHy do we put it as  public doc? Is it going to be changed?

Reply:
Accidentally did it, replaced it with internal.

Also, added comment on the top of the file, that it will be removed.